Monika After Story/game/script-topics.rpy

```diff
@@ -13041,6 +13041,73 @@ label monika_auroras:
+
+default persistent._mas_pm_likes_board_games = None
+# True if player likes board games, false if not
+
+label monika_boardgames:
+    m 1eua "Say, [player], you like playing video games, right?"
+    m 2rsc "Well, I assume you do...{w=1} {nw}"
+    extend 2eud " I don't know if many people would play a game like this one if they weren't at least a little into video games in the first place."
+    m 2eua "But I was wondering, what do you think of board games?"
+    m 4eub "As you probably know, board games have been around since very early in our history. And video games most likely wouldn't be a thing today if it weren't for them."
+    m 1rksdlb "...I guess that means I probably wouldn't have existed either, if our ancestors didn't play board games in their time.{w=0.3} {nw}"
+    extend 1hksdlb "ahaha~"
+    m 1euc "I never really got the occasion to play a lot of board games myself, so I'm a little intrigued by them."
+    if renpy.seen_label('unlock_chess') or renpy.seen_label('game_chess') :
+        m 1ruc "Well, except for chess, of course. And a few card games as well..."
+    else :
+        m 1rud "Well, I did try out a few basic card games."
+        m 1hua "As well as a little something I've been working on... I'm keeping it a surprise, though!"
+    m 1etc "But I can't help but think... Those are pretty basic and well known games, right? I'm sure most people have played a game like that at least once in their lives."
+    m 3esa "Even if I never got too much into this before, I figured there had to be a wider variety of games out there and,{w=0.2} well,{w=0.2} I got curious and made a bit of research to see for myself."
+    m 3wud "And, as it turns out...{w=0.5} This is actually a pretty big market!"
+    m 1eub "Board games tend to be a little overshadowed by video games nowadays, but there seems to be a lot of people who are really passionate about them anyway."
+    m "Some of those people will want to play any game they can, and might even spend a lot of money into buying a lot of different games."
+    m 1lksdla "...Not that I'm judging, of course. It's nice that they are having fun."
+    m 1esc "But since some people are willing to spend a lot of money into those things, it means that some board games can get really expensive."
+    m 3esa "With figures, pretty illustrations, high quality materials..."
+    m "Those kinds of things."
+    m 2lsc "Though, as fancy as those particular games look, one might wonder if they aren't meant for collection more than actually playing, at this point."
+    m 2euc "Speaking of collecting..."
+    m 2eud "Maybe you've heard of those card games where you collect as much cards as possible and then play against someone else's cards?"
+    m 2dtc "What was the name again? Card...   ...Trading..."
+    m 4eud "Ah, right, Trading Card Games, or TCG for short."
```

Comment:
I didn't know they had an acronym. If this is a niche fact, I'm not sure Monika would know this either.